AEPConcierge/Sources/Services/ConciergeChatService.swift

```diff
@@ -221,20 +221,6 @@ extension ConciergeChatService: URLSessionDataDelegate {
                 let handle = try JSONDecoder().decode(TempHandle.self, from: handleData)
                 if let handler = self.onChunkHandler,
                    let payload = handle.handle.first?.payload.first {
-                    if let resp = payload.response {
-                        let text = resp.message
-                        let srcCount = resp.sources?.count ?? 0
-                        let suggCount = resp.promptSuggestions?.count ?? 0
-                        Log.debug(label: LOG_TAG, "SSE chunk: state=\(payload.state ?? "n/a"), textLen=\(text.count), sources=\(srcCount), suggestions=\(suggCount)")
-                        if payload.state == Constants.StreamState.COMPLETED {
-                            if let data = try? JSONEncoder().encode(resp),
-                               let json = String(data: data, encoding: .utf8) {
-                                Log.debug(label: LOG_TAG, "SSE final response JSON: \(json)")
-                            }
-                        }
-                    } else {
-                        Log.debug(label: LOG_TAG, "SSE chunk: state=\(payload.state ?? "n/a") (no response)")
-                    }
```

Comment:
thank you for moving this - i was just looking at doing the same while i was working on the configuration changes :)